pkg/plugin/util/util.go

```diff
@@ -110,3 +116,111 @@ func SelfLinkToCRDName(selfLink string) string {
 		return components[pluralIndex]
 	}
 }
+
+func GetClient(config *rest.Config, logger logrus.FieldLogger) (*kubernetes.Clientset, error) {
+	var err error
+	if config == nil {
+		config, err = rest.InClusterConfig()
+		if err != nil {
+			logger.WithError(err).Errorf("Failed to get k8s inClusterConfig")
+			return nil, errors.Wrap(err, "could not retrieve in-cluster config")
+		}
+	}
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		logger.WithError(err).Error("Failed to get k8s clientset from the given config")
+		return nil, err
+	}
+	return clientset, nil
+}
+
+func CheckStorageClassMapping(storageClassMapping map[string]string, configData map[string]string, logger logrus.FieldLogger) error {
```

Comment:
Add comments to explain what this function does.

Reply:
I removed this CheckStorageClassMapping method as there is no need to do the check here. In the previous implementation there should only be one configMap for the specified labels. We can return configMaps.Items[0].Data directly, which is a map contains all the old-new storage class pairs.